pkg/configuration/config.go

```diff
@@ -65,6 +65,12 @@ func Setup() {
 		}
 	}
 
+	// This may be different from `UserConfigDir` if `--configuration` was provided
+	configDir := filepath.Dir(UserConfigFile)
+	if !utils.Exists(configDir) {
+		utils.HandleError(fmt.Errorf("Configuration file directory does not exist: %s", configDir))
```

Comment:
For consistency with our other errors
```suggestion
		utils.HandleError(fmt.Errorf("Configuration file directory does not exist %s", configDir))
```

Reply:
Ah, good catch